.github/workflows/update_sdk.yml

```diff
@@ -0,0 +1,132 @@
+name: Update SDK
+
+on:
+  schedule:
+    - cron: "0 * * * *" # Run each hour
```

Comment:
IMO: Daily is enough.  Either is fine.

Reply:
Fixed in https://github.com/ynkdir/py-win32more/pull/66/commits/493f3a7321f746091e3b0bf4b12e345628c52106